crates/astria-conductor/src/executor/mod.rs

```diff
@@ -272,25 +328,23 @@ impl Executor {
     /// Panics if the `u32` underlying the celestia block variance tracked in the state could
     /// not be converted to a `usize`. This should never happen on any reasonable architecture
     /// that Conductor will run on.
-    fn calculate_max_spread(&self) -> usize {
-        usize::try_from(self.state.borrow().celestia_block_variance())
+    fn calculate_max_spread(&mut self) -> usize {
```

Comment:
This actually begs the question: why batch-request all these blocks in the first place? We could just as well request the parent-block from the rollup as we go instead of maintaing a cache.